README.md

```diff
@@ -49,7 +55,17 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town = {
+  residents: ["Maurice", "Belle", "Gaston"],
+  castle: {
+    num_rooms: 47,
+    residents: "Robby Benson",
+    guests: []
+  }
+}
+
+residents_castle[:key2] = residents.delete(:key1)
+
```

Comment:
First step here is figuring out how to access the values in question. "Belle" is inside an array `residents` which is inside a hash `town.` So to access that value, you'll need to do something like this...

```rb
town["residents"][1]
```
```